pom.xml

```diff
@@ -507,6 +507,10 @@
                             <name>listener</name>
                             <value>com.shaft.tools.listeners.AlterSuiteListener,com.shaft.tools.listeners.SuiteListener,com.shaft.tools.listeners.InvokedMethodListener,com.shaft.tools.listeners.CucumberFeatureListener</value>
                         </property>
+                        <property>
+                            <name>reporter</name>
+                            <value>org.testng.reporters.XMLReporter:generateTestResultAttributes=true,generateGroupsAttribute=true</value>
+                        </property>
```

Comment:
I would love to remove this section as well, but I don't know how to do it :D So, let's keep it as is and please plan to remove it in the future if possible.